src/main/java/mekanism/common/item/interfaces/IFreeRunnersItem.java

```diff
@@ -0,0 +1,101 @@
+package mekanism.common.item.interfaces;
+
+import com.mojang.serialization.Codec;
+import io.netty.buffer.ByteBuf;
+import mekanism.api.IIncrementalEnum;
+import mekanism.api.annotations.NothingNullByDefault;
+import mekanism.api.text.EnumColor;
+import mekanism.api.text.IHasTextComponent;
+import mekanism.api.text.ILangEntry;
+import mekanism.common.Mekanism;
+import mekanism.common.MekanismLang;
+import mekanism.common.integration.curios.CuriosIntegration;
+import net.minecraft.network.chat.Component;
+import net.minecraft.network.codec.ByteBufCodecs;
+import net.minecraft.network.codec.StreamCodec;
+import net.minecraft.util.ByIdMap;
+import net.minecraft.util.StringRepresentable;
+import net.minecraft.world.entity.EquipmentSlot;
+import net.minecraft.world.entity.LivingEntity;
+import net.minecraft.world.item.ItemStack;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Locale;
+import java.util.function.IntFunction;
+import java.util.function.Predicate;
+
+public interface IFreeRunnersItem {
```

Comment:
I don't really see the need for the interface?

Reply:
Jetpacks also have their own interface to make more clear that operation is happening on some jetpack rather that concrete type, in my opinion this increases clarity and consistency over code base.

If you think that it is REALLY unnecessary I will remove it and make it operate on ItemFreeRunners.